### PR TITLE
added new covid care project

### LINKED ISSUE #511: Add Covid Care
https://github.com/j-j-gajjar/covid-care

I want to add this in Health and fitness

### PR DESCRIPTION
**Pull request by Maheshwari-Love**
Added Covid Care project 

close #511

### CODE REVIEW

**Comment by Maheshwari-Love** [comment]
@ashitaprasad please review this pr

File: README.md [hunk 1]
```diff
@@ -194,6 +194,7 @@ All the projects added in this project are featured in [fluttergems.dev](flutter
 | Open Food Facts | [Link](https://github.com/openfoodfacts/smooth-app) | App to scan, discover & compare over 3 million food products. |
 | Food Delivery App UI | [Link](https://github.com/Tarikul711/flutter-food-delivery-app-ui) | Food ordering & delivery application UI. |
 
+
```
Comment (ashitaprasad) [inline README.md:197]: Do not add new lines unnecessarily

**Comment by ashitaprasad** [comment]
@Maheshwari-Love Remove the extra new lines you have added.

**Comment by Maheshwari-Love** [comment]
@ashitaprasad kindly check i have resolved the conflicts and removed extra lines

**Comment by ashitaprasad** [comment]
There are still empty lines visible.
Closing this PR, raise a new one only with the added project.

**Pull request closed without merging**